Update doc to take into account the new Configuration update behavior (#115)
project-akri/akri-docs: docs/user-guide/customizing-an-akri-installation.md

1. Using the `helm upgrade` command 
2. [Generating, modifying and applying a custom Configuration](customizing-an-akri-installation.md#generating-modifying-and-applying-a-configuration)

> You can only modify a subset of the Configuration this way, only the broker properties as well as the capacity can be modified this way, for any other modification, you need to delete and
> reapply the Configuration
> 

Reviewer: ```suggestion
Note: Only the broker properties and capacity of an applied configuration should be modified, for any other modification, you need to delete and reapply the Configuration.
```
